src/components/SearchResultItem.tsx

```diff
@@ -19,16 +21,23 @@ export class SearchResultItem extends React.Component<SearchResultItemProps, Sea
 
     render() {
         let iconPath = this.props.data.iconUrl;
+        let parentText = this.props.pathToItem[this.props.pathToItem.length - 2].text;
+        let categoryText = this.props.pathToItem.find(item => item.itemType === "category").text;
```

Comment:
Lack of documentation, please improve on this. This section is growing and will soon become incomprehensible if we don't document it properly. What does `pathToItem.length - 2` mean? How is `categoryText` derived? It cannot be understood unless we have proper document with examples here.